### ISSUE TITLE
Update to Julia 1.0

### ISSUE THREAD

**Issue by bkamins** [nalimilan/FreqTables.jl#27]
@nalimilan I do not want to start pushing PRs migrating this package to Julia 1.0, before I understand your plans regarding this. Most of things that fail should be fixable by femtocleaner I guess (and `NamedArrays` on master are ready/almost ready for Julia 1.0).

Please let me know how you want to proceed and how I could help (I really like this package 😄).

**Comment by nalimilan** [comment]
Please feel free to go ahead and do anything necessary to get the tests to pass on 1.0. Hopefully Femtocleaner will do most of the work.

**Comment by bkamins** [comment]
Yes, but I guess it is you who should initiate Femtocleaner on the repo? Or I am missing something?

**Comment by nalimilan** [comment]
AFAICT it's enabled already?

**Comment by bkamins** [comment]
But I saw in `DataFramesMeta.jl` that Femtocleaner automatically created a PR. Do you know how to trigger this? Then I would review what would be left broken.

**Comment by nalimilan** [comment]
You just have to bump the minimum Julia version in REQUIRES and open a PR, and it should clean everything it can.

**Comment by bkamins** [comment]
OK. I will do it when `NamedArrays` are tagged with a new release.

**Comment by nalimilan** [comment]
See https://github.com/nalimilan/FreqTables.jl/pull/28 for a first step.
